Editor/GameLiftClientSettingsEditor.cs

```diff
@@ -49,14 +45,43 @@ public override void OnInspectorGUI()
             }
         }
 
-        private void EditLocalMode(GameLiftClientSettings targetSettings)
+        private void OnEnable()
```

Comment:
Could we rename `OnEnable`? From reading code it isn't apparent to me what exactly this is enabling as I'm not familiar with Unity

Reply:
OnEnable is a standard callback in Unity that's triggered when the object has loaded